README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot]()
+
+
+## Built With
+
+- VSCODE
+- Github
+- Git
+
+## Getting Started
+
+
+## Authors
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
## Changes Required:

⚠️ You're missing the `MIT.md` file, that gives details about your project's license. Please modify it with your information and add it to the root folder of your project. You can find the template on this repo: https://github.com/microverseinc/readme-template . 👍